wallet/wallet_test.go

```diff
@@ -95,7 +169,7 @@ func TestWallet(t *testing.T) {
 	}
 
 	// try funding the transaction, expect it to fail since the outputs are immature
-	_, err = w.FundTransaction(&txn, initialReward, false)
+	_, err := w.FundTransaction(&txn, initialReward, false)
 	if err != wallet.ErrNotEnoughFunds {
```

Comment:
This needs to be updated now to use `errors.Is`.

Reply:
woopsie